mobile/src/main/java/org/openhab/habdroid/ui/OpenHABMainActivity.java

```diff
@@ -551,8 +560,75 @@ public void onSuccess(Call call, int statusCode, Headers headers, String respons
     }
 
     public void onError(String error) {
-        Toast.makeText(getApplicationContext(), error,
-                Toast.LENGTH_LONG).show();
+        showMessageToUser(error, "dialog", 5);
+    }
+
+    /**
+     * Shows a message to the user.
+     * You might want to send two messages: One detailed one with logLevel 0
+     * and one simple message with 4
+     *
+     * @param message message to show
+     * @param messageType can be dialog, snackbar or toast (deprecated)
+     * @param logLevel 0 when debug is enabled
+     *                 1 when remote url is configured
+     *                 2 when local url is configured
+     *                 4 when debug is disabled
+     *                 5 always
+     */
+    public void showMessageToUser(String message, String messageType, int logLevel) {
```

Comment:
mesageType should be an int or enum, shouldn't it?

Reply:
Or a constant?

Reply:
I created constant int for that. Thanks.